@navikt/core/react/src/stepper/Step.tsx

```diff
@@ -0,0 +1,61 @@
+import cl from "classnames";
+import React, { forwardRef, useContext } from "react";
+import { StepperContext } from ".";
+import { Label, OverridableComponent } from "..";
+
+export interface StepperStepProps
```

Comment:
![Screenshot 2022-03-11 at 11 37 28](https://user-images.githubusercontent.com/26967723/157852188-0c30ffd4-1947-42fd-98ac-b479c2776cef.png)

Ser litt forskjellige variasjoner for "paginering" av stepper. Er dette versjonene som er bestemt nå?

Reply:
Om så vil kanskje next/prev fungere som egne knapper, eks: `<Stepper.Previous/> / <Stepper.Next/>`

Reply:
Jeg stemmer for at vi implementerer den funksjonaliteten i en annen PR.